projects/plugins/jetpack/modules/search/customberg/components/app-wrapper/index.jsx

```diff
@@ -74,7 +74,14 @@ export default function AppWrapper() {
 	const { isLoading } = useSiteLoadingState();
 
 	return (
-		<div className="jp-search-configure-app-wrapper">
+		<div
+			/* translators: accessibility text for the widgets screen content landmark region. */
+			aria-label={ __( 'Jetpack Search customization preview', 'jetpack' ) }
+			className="jp-search-configure-app-wrapper"
+			// className="interface-interface-skeleton__content"
```

Comment:
probably we'd want to remove the commented line.

Reply:
Ah, good catch!